okhttp/src/main/java/io/grpc/okhttp/OkHttpClientStream.java

```diff
@@ -190,11 +190,14 @@ protected void sendFrame(WritableBuffer frame, boolean endOfStream, boolean flus
     }
 
     synchronized (lock) {
+      if (cancelSent) {
```

Comment:
Is the assumption that the stream will be closed properly by the cancel processing?